neurodamus/node.py

```diff
@@ -270,6 +271,12 @@ def __init__(self, config_file, options=None):
             self._spike_populations = []
             Nd.execute("cvode = new CVode()")
             SimConfig.init(config_file, options)
+            if SimConfig.coreneuron:
+                CoreConfig.output_root = SimConfig.output_root
+                CoreConfig.datadir = SimConfig.coreneuron_datadir
+                # Instantiate the CoreNEURON artificial cell object which is used to fill up
+                # the empty ranks. This need to be done before the circuit is finitialized
+                CoreConfig.instantiate_artificial_cell()
```

Comment:
Could this be moved into `SimConfig.init`? Would maybe be a bit more logical

Reply:
`SimConfig` gets instantiated during validation of the configuration read by neurodamus and before MPI is initialized/imported. Since CoreConfig imports MPI we would like to keep it separated from SimConfig